.github/workflows/ci.yml

```diff
@@ -135,7 +143,9 @@ jobs:
 
   microsite:
     name: Publish Docs Microsite
-    if: startsWith(github.ref, 'refs/tags/v') || (github.event_name == 'workflow_dispatch' && github.event.inputs.publishMicrosite == 'true')
+    if:
+      startsWith(github.ref, 'refs/tags/v') ||
+      github.event.inputs.publishMicrosite
```

Comment:
same as above, removed the event_name check out of redundancy.

I didn't make the microsite get published on the schedule as schedules only run on the default branch, meaning we couldn't publish the site from `series/0.17` that way.